apps/heft/src/cli/actions/BuildAction.ts

```diff
@@ -145,6 +154,18 @@ export interface ICompileStageProperties {
   copyStaticAssetsConfiguration: ICopyStaticAssetsConfiguration;
 }
 
+/**
+ * @public
+ */
+export interface IBundleStageProperties {
+  /**
+   * A path to a Webpack configuration JS file. If this isn't specified, and a Webpack
+   * configuration isn't specified via another plugin, Webpack won't be run.
+   */
+  webpackConfigFilePath?: string;
+  webpackConfiguration?: webpack.Configuration;
```

Comment:
Can both of these be specified together?

Reply:
I'll leave a comment.